test/__test__/userAdmin.test.js

```diff
@@ -0,0 +1,28 @@
+const request = require('supertest');
+const app = require('../../app');
+const User = require('../../app/models').users;
+const mockUserAdmin = require('../mockUserAdmin');
+const helper = require('../../app/helpers/helpers');
+
+describe('Should get a token', () => {
```

Comment:
Change the description, the same with the other tests.
Should have a general description of file you are testing.